docs/auth/server/server.js

```diff
@@ -0,0 +1,81 @@
+const createError = require('http-errors');
+const express = require('express');
+const morgan = require('morgan');
+const mustacheExpress = require('mustache-express');
+const session = require('express-session');
+const http = require('http');
+
+const app = express();
+
+app.engine('mustache', mustacheExpress());
+app.set('view engine', 'mustache');
+app.set('views', __dirname + '/views');
+
+app.use(morgan('dev'));
+app.use(express.json());
```

Comment:
do we ever use this?

Reply:
i thought it was needed for `body` to be parsed on post but that was wrong wrong wrong wrong wrong